standup_notes/__main__.py

```diff
@@ -124,6 +164,36 @@ def iterate_weekday(day: date, func: Callable[[date], date]) -> date:
         return iterate_weekday(next_day, func)
 
 
+def delete_notes(amount):
```

Comment:
Could you add what param type "amount" is?

Reply:
the amount is the date value entered, I will change the variable name